x-pack/legacy/plugins/maps/public/connected_components/map/mb/view.js

```diff
@@ -22,12 +25,8 @@ import { FeatureTooltip } from '../feature_tooltip';
 import { DRAW_TYPE } from '../../../actions/map_actions';
 import { createShapeFilterWithMeta, createExtentFilterWithMeta } from '../../../elasticsearch_geo_utils';
 import chrome from 'ui/chrome';
-
-function relativeToAbsolute(url) {
-  const a = document.createElement('a');
-  a.setAttribute('href', url);
-  return a.href;
-}
+import { spritesheet } from '@elastic/maki';
+import sprites from '@elastic/maki/dist/sprite@1.png';
```

Comment:
Is there a way to see if the device is retina display and load `@2.png` if it is?